content/migrations/overview/programmatically-importing-repositories.md

```diff
@@ -9,7 +9,7 @@ type: tutorial
 
 ## About programmatic import of repositories
 
-In the following guide, you can learn how to programatically run "source and history" migrations of Git repositories to {% data variables.product.prodname_dotcom_the_website %}. Different options are available depending on where the repository is stored.
+In the following guide, you can learn how to programmatically run "source and history" migrations of Git repositories to {% data variables.product.prodname_dotcom_the_website %}. Different options are available depending on where the repository is stored.
```

Comment:
#### Before
https://docs.github.com/en/migrations/overview/programmatically-importing-repositories#about-programmatic-import-of-repositories
<img width="736" alt="image" src="https://github.com/github/docs/assets/2119212/a179c5bb-05d3-4b28-acb5-ed4bd9c22c67">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/migrations/overview/programmatically-importing-repositories#about-programmatic-import-of-repositories
<img width="732" alt="image" src="https://github.com/github/docs/assets/2119212/48b33bf4-70b4-46d7-b918-cecaaef73825">